test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -650,6 +666,148 @@ private HapiApiSpec happyPathWithRepeatedMetadata() {
 				);
 	}
 
+	private HapiApiSpec wipeHappyPath() {
+		return defaultHapiSpec("wipeHappyEnd")
+				.given(
+						newKeyNamed(SUPPLY_KEY),
+						newKeyNamed("wipeKey"),
+						newKeyNamed("treasuryKey"),
+						newKeyNamed("accKey"),
+						cryptoCreate(TOKEN_TREASURY).key("treasuryKey"),
+						cryptoCreate("account").key("accKey"),
+						tokenCreate(NFT)
+								.tokenType(NON_FUNGIBLE_UNIQUE)
+								.supplyType(TokenSupplyType.INFINITE)
+								.supplyKey(SUPPLY_KEY)
+								.initialSupply(0)
+								.treasury(TOKEN_TREASURY)
+								.wipeKey("wipeKey"),
+						tokenAssociate("account", NFT),
+						mintToken(NFT, List.of(ByteString.copyFromUtf8("memo"), ByteString.copyFromUtf8("memo2"))),
+						cryptoTransfer(
+								movingUnique(2L, NFT).between(TOKEN_TREASURY, "account")
+						)
+				)
+				.when(
+						wipeTokenAccount(NFT, "account", List.of(1L)).via("wipeTxn")
```

Comment:
This is actually a bug. We should not be able to `wipe` a serial number for the specified account if the account does not own the `serialNumber`.